plugins/sentinelone/komand_sentinelone/tasks/monitor_logs/schema.py

```diff
@@ -44,7 +44,7 @@ class MonitorLogsOutput(insightconnect_plugin_runtime.Output):
   "title": "Logs",
   "description": "List of activity, device control event, and threat logs within the specified time range",
   "items": {
-    "type": "object"
+    "$ref": {}
```

Comment:
This schema change needs reverted back, it breaks the C2C tasks from memory!

Reply:
Thanks! That's why my unit tests were failing!